### ISSUE TITLE
`Invalid VNode`  error with v-for

### ISSUE THREAD

**Issue by husayt** [Planning-nl/vugel#3]
Is there anything one needs to be aware of with v-for?
```
<compiler compiler="vugel">
    <container>  
      <text v-for="a in 10" :key="a" :x="110" :y="10">{{a}}</text>
    </container>
</compiler>
```
I have spent some time trying to get above working. It works without v-for, but with v-for I get the following error:
```
Invalid VNode type: Symbol(Fragment) (symbol) 
  at <TestComp> 
  at <Anonymous> 
```

Edit: I guess this is related to custom-block implementation, rather than Vugel itself. With codepen example v-for works fine, but when I try with vite and the custom block implemented [here](https://github.com/RobbinBaauw/CustomCompilerBlock) I get above exception. 

Custom block is discussed here #2

**Comment by RobbinBaauw** [comment]
Yeah, the custom block impl may not work great. If you can reproduce this in codepen there would be an actual bug. `v-for` etc should just work normally.
